src/js/src/store/modules/search.store.js

```diff
@@ -13,6 +13,11 @@ const initialState = () => ({
     imgSearch:false,
     urlSearch:false
   },
+  futureSolrSettings:{
+    grouping:false,
+    imgSearch:false,
+    urlSearch:false
+  },
```

Comment:
Nice getting this in here - gave you some more flexibility to separate things 👍

Reply:
:+1: